src/openrct2/ride/Vehicle.cpp

```diff
@@ -818,6 +818,25 @@ uint16_t Vehicle::GetTrackProgress() const
     return vehicle_get_move_info_size(TrackSubposition, track_type);
 }
 
+void Vehicle::MoveRelativeDistance(int32_t distance)
+{
+    remaining_distance += distance;
+
+    SetUpdateFlag(VEHICLE_UPDATE_FLAG_SINGLE_CAR_POSITION);
+    UpdateTrackMotion(nullptr);
+    ClearUpdateFlag(VEHICLE_UPDATE_FLAG_SINGLE_CAR_POSITION);
+
+    const Ride* curRide = GetRide();
```

Comment:
I'm pretty sure you don't need to do any of this  moveInfo changes. UpdateTrackMotion should take care of it.

Reply:
Yes you're right, I'll remove it. :)